src/App.js

```diff
@@ -1,15 +1,104 @@
 import SearchBar from './components/SearchBar/SearchBar';
 import Users from './components/Users/Users';
 import './App.css';
+import { useEffect, useState } from 'react';
+import NoResult from './components/NoResult/NoResult';
+import Loading from './components/Loading/Loading';
+import Error from './components/Error/Error';
+
+const API = "https://users-app-backend.onrender.com/users"
 
 function App() {
+  const [ users, setUsers ] = useState([]);
+  const [ searchInput, setSearchInput ] = useState("");
+  const [ expanded, setExpanded ] = useState([]);
+  const [ loading, setLoading ] = useState(false);
+  const [ error, setError ] = useState("");
+
   // TODO: Fetch data here
+  useEffect(()=> {
+    async function fetchData() {
+      try {
+        setError("");
+        setLoading(true);
+        const response = await fetch(`${API}`);
+        const json = await response.json();
+        const { data, error } = json;
+        if(response.ok) {
+          setUsers(data);
+          setLoading(false);
+        } else {
+          setError(error);
+          setLoading(false);
+        }
+        
+      } catch (error) {
+        setError(error.meassage);
```

Comment:
might be a typo here (`meassage` instead of `message`)

Reply:
Thank you. That was a typo.